database/pgtsv_to_json

```diff
@@ -3,6 +3,29 @@
 # Usage: pgtsv_to_json COLUMN_NAME1:TYPE1 COLUMN_NAME2:TYPE2 ...
 
 import json, sys, csv, re
+from datetime import datetime
+
+def timestamp(timestamp_str):
+    """Given a timestamp string, return a timestamp string in ISO 8601 format to emulate
+    Postgres 9.5's to_json timestamp formatting. The given timestamp is assumed UTC.
+
+    Time zones are not supported. http://bugs.python.org/issue6641
+
+    Examples:
+
+        >>> timestamp('2016-06-17 20:10:38')
+        '2016-06-17T20:10:38+00:00'
+
+        >>> timestamp('2016-06-17 20:10:37.9293')
+        '2016-06-17T20:10:37.929300+00:00'
+
+    """
+
+    try:
+        parsed = datetime.strptime(timestamp_str, '%Y-%m-%d %H:%M:%S.%f')
+    except ValueError:
+        parsed = datetime.strptime(timestamp_str, '%Y-%m-%d %H:%M:%S')
+    return parsed.isoformat() + '+00:00'
```

Comment:
1 sounds better. But we are probably over-engineering here. I'd simply output verbatim whatever psql feeds us. Chances are the downstream app would actually prefer verbatim. For the current main consumer of ES, it supports a variety of formats already: https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-date-format.html